src/Common/tests/System/Collections/ISet.Generic.Tests.cs

```diff
@@ -419,6 +419,7 @@ public void ISet_Generic_ExceptWith_Itself(int setLength)
 
         [Theory]
         [MemberData(nameof(ValidCollectionSizes))]
+        [SkipOnTargetFramework(TargetFrameworkMonikers.NetFramework)] //Throws InvalidOperationException, issue: dotnet/corefx #16796
```

Comment:
In a few cases below, the issue number is in the string argument to SkipOnTargetFramework... could the same be done here, rather than having it in a comment?